backends/arm/test/common.py

```diff
@@ -5,10 +5,85 @@
 # LICENSE file in the root directory of this source tree.
 
 import os
+import shutil
+import subprocess
 import tempfile
 
+import pytest
+
+import torch
+
 from executorch.backends.arm.arm_backend import ArmCompileSpecBuilder
 
+_enabled_options: list[str] = []
+
+# ==== Pytest hooks ====
+
+
+def pytest_addoption(parser):
+    parser.addoption("--arm_quantize_io", action="store_true")
+    parser.addoption("--arm_run_corstone300", action="store_true")
```

Comment:
This is same as before right? I.e. we did run FVP on non-trunk jobs. Your comment says otherwise. I can check too

Reply:
This is the first patch that integrates FVP in testing